interfaces/builtin/camera.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const cameraConnectedPlugAppArmor = `
+/dev/video0 rw,
+`
+
+const cameraConnectedPlugSecComp = `
+read
+write
+ioctl
+`
+
+// NewCameraInterface returns a new "camera" interface.
+func NewCameraInterface() interfaces.Interface {
+	return &commonInterface{
+		name: "camera",
+		connectedPlugAppArmor: cameraConnectedPlugAppArmor,
+		connectedPlugSecComp:  cameraConnectedPlugSecComp,
+		reservedForOS:         true,
```

Comment:
I can spell it out but since false is default (zero go value) I left it out. I'm happy to put it back if you want to